### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by s12chung**
removed features:
- dependencies option - not needed after cleaning up `csv_row_model`

i'd read each commit (last commit moves around lots of stuff)

### CODE REVIEW

File: spec/inherited_class_var_spec.rb [hunk 1]
```diff
@@ -34,32 +34,6 @@ class InheritedBaseClass; end
...
-        end
-
-        it 'merge options', skip: true do
-          Parent.merge_inherited_hash(test1: { foo: 'test1' })
-          expect(Parent.inherited_hash[:test1]).to eql({ foo: 'test1' })
-
-          ClassWithFamily.merge_inherited_hash(test1: { bar: 'test2' })
-          expect(ClassWithFamily.inherited_hash[:test1]).to eql({ foo: 'test1', bar: 'test2' })
-        end
-      end
```
Comment (s12chung) [inline spec/inherited_class_var_spec.rb]: i don't think this is needed?

i think you were confused about what the caching does. clearing the cache won't change the values of the `inherited_class_var`, it just forces a recalculation in the block: https://github.com/FinalCAD/inherited_class_var/blob/e6a20ce15933dbec12e1a3be8a2c2b6fbcf95b41/lib/inherited_class_var.rb#L72-L76

i wrote a comment to explain why i wrote the caching code here: https://github.com/FinalCAD/inherited_class_var/pull/3/commits/9909414127e1deee9505f4399a6dc9707b0056f4

File: spec/inherited_class_var_spec.rb [hunk 2]
```diff
@@ -34,32 +34,6 @@ class InheritedBaseClass; end
         expect(Parent.inherited_hash).to eql(test1: 'test1', test2: 'test2')
         expect(Parent.inherited_hash.object_id).to eql Parent.inherited_hash.object_id
       end
-
-      context 'with cache cleared' do
-        before do
-          # NOTE deep_clear_class_cache doesn't work here...
```
Comment (s12chung) [inline spec/inherited_class_var_spec.rb:46]: i think you were confused about what the caching does. clearing the cache won't change the values of the `inherited_class_var`, it just forces a recalculation in the block: https://github.com/FinalCAD/inherited_class_var/blob/e6a20ce15933dbec12e1a3be8a2c2b6fbcf95b41/lib/inherited_class_var.rb#L72-L76

i wrote a comment to explain why i wrote the caching code here: https://github.com/FinalCAD/inherited_class_var/pull/3/commits/9909414127e1deee9505f4399a6dc9707b0056f4

so i don't think this code is needed.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
-
-begin
-  require 'pry'
-rescue LoadError
-end
-
+Bundler.require(:default, :test)
 Coveralls.wear!
 
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
```
Comment (joel) [inline spec/spec_helper.rb]: useless...

File: spec/inherited_class_var_spec.rb (hunk 2 above)
Comment (joel) [inline spec/inherited_class_var_spec.rb:46]: Thanks for the comment

File: spec/spec_helper.rb (hunk 3 above)
Comment (s12chung) [inline spec/spec_helper.rb]: lol, we talked about this before---i prefer it, but it seems like it bugs ya. i'll remove it just for you ;)

Comment (joel) [inline spec/spec_helper.rb]: Thanks ;-)

**Pull request merged**